pkg/ebpf/tracee.go

```diff
@@ -1576,6 +1592,18 @@ func (t *Tracee) getSelfLoadedPrograms(kprobesOnly bool) map[string]int {
 						continue
 					}
 				}
+				key := probeMapKey{
+					programName: p.GetProgramName(),
+					probeType:   p.GetProbeType(),
+				}
+
+				_, found := uniqueHooksMap[key]
+				if found { // Already counted this hook
+					continue
+				}
+
+				uniqueHooksMap[key] = struct{}{}
+
 				log(definition.GetName(), p.GetProgramName())
```

Comment:
I know this is not part of the current code, but shouldn't we use the `logger.Infow` here?

Reply:
This is a change done by a previous maintainer so not sure the motivation